src/main/resources/application-worker.yaml

```diff
@@ -30,3 +30,8 @@ rhsm-subscriptions:
     internal:
       url: http://${CLOUDIGRADE_INTERNAL_HOST}:${CLOUDIGRADE_INTERNAL_PORT}/internal/api/cloudigrade/v1
       presharedKey: ${CLOUDIGRADE_PSK:}
+  tally-summary-producer:
+    back-off-initial-interval: ${TALLY_SUMMARY_PRODUCER_BACK_OFF_INITIAL_INTERVAL:1s}
```

Comment:
Need to update the `templates/` and `deploy/rhsm-clowdapp.yaml` files with these new env vars

https://github.com/RedHatInsights/rhsm-subscriptions/blob/ac5bab799c8c93ea7dc42a830bddf4a81a16a717/templates/rhsm-subscriptions-worker.yml

https://github.com/RedHatInsights/rhsm-subscriptions/blob/ac5bab799c8c93ea7dc42a830bddf4a81a16a717/deploy/rhsm-clowdapp.yaml

Also, as we've been adding new env vars, we've started converging on a pattern where we define env vars with defaults at the top of the file first - so like at the top of this yaml file you'd have `TALLY_SUMMARY_PRODUCER_BACK_OFF_INITIAL_INTERVAL=1s`, then here you would just put `${TALLY_SUMMARY_PRODUCER_BACK_OFF_INITIAL_INTERVAL}` (like `CLOUDIGRADE_INTERNAL_HOST` in this same file)

Reply:
oh makes sense, thanks